server/Cargo.toml

```diff
@@ -8,6 +8,8 @@ trustify-auth = { path = "../common/auth", features = ["actix"] }
 trustify-graph = { path = "../graph" }
 trustify-common = { path = "../common"}
 trustify-infrastructure = { path = "../common/infrastructure" }
+trustify-entity = { path = "../entity"}
+trustify-formats = { path = "../ingestors"}
```

Comment:
I think this should be named `trustify-formats` to align the directory name?

Reply:
I'm following what we already have into the `main` branch. The `main` branch has a directory `ingestors/` whose cargo module name is `trustify-formats`.

https://github.com/trustification/trustify/blob/a84adf048dfe98df3b3a54c12ac84a602e935934/ingestors/Cargo.toml#L2

I'm probably missing something, what should I change? the directory name or the line 12 `trustify-formats = { path = "../ingestors"}`

Reply:
I don't know TBH. I was just confused by the fact I didn't see any use of `trustifty_ingestors`, but instead there is a `trustify_formats` crate which doesn't seem to exist in the filesystem.

I think names should be aligned unless there are good reasons not to. And if that's not the case, we should fix it.

It looks like this is all about ingesting, so naming it `trustify-ingestors` sounds reasonable to me.

Reply:
I fully agree with you. I renamed the package name to `trustify-ingestors` for it to be aligned to the directory name. I assume it was a mistake to have it that way (main branch) in the first place